### PR TITLE
Status

### PR DESCRIPTION
**Pull request by Miminiverse**
Update participant array with userID so user can submit their participation status

### CODE REVIEW

File: src/controllers/session.ts [hunk 1]
```diff
@@ -112,67 +116,95 @@ const deleteSession = async (req: Request, res: Response) => {
...
-            user: {
-              userInfo: req.user.userId,
-              userStatus,
-            },
-          },
-        },
-      },
-      { new: true }
+
+  let event;
```
Comment (cdpaiva) [inline src/controllers/session.ts:139]: This variable is not being used anywhere, we could also remove the commented code.

File: src/controllers/session.ts [hunk 2]
```diff
@@ -112,67 +116,95 @@ const deleteSession = async (req: Request, res: Response) => {
...
-      }
-      event = await scheduleEvent({
-        summary: sessionType,
-        start: sessionStart,
-        end: sessionEnd,
-        email: req.user.email,
-        description: sessionCreator,
-      });
-    }
+  return res.status(200).json({ session, event });
```
Comment (cdpaiva) [inline src/controllers/session.ts:183]: `event` could also be removed from here

File: src/controllers/session.ts [hunk 3]
```diff
@@ -112,67 +116,95 @@ const deleteSession = async (req: Request, res: Response) => {
...
+};
 
-    return res.status(201).json({ newSessionUser, event });
+const getStatus = async (req: Request, res: Response) => {
```
Comment (cdpaiva) [inline src/controllers/session.ts:186]: This method is fine, but we could also check the user status in the Front-end. In there, we can directly check the currently logged userId against the list of userIds of each session.

File: src/controllers/session.ts (hunk 2 above)
Comment (Miminiverse) [reply src/controllers/session.ts:183]: The event and the commented out code is used to for the  Google calendar, but I was having some issues and I'm still working on it, I need to figure out why it doesn't send out the schedule

**Pull request merged**